### PR TITLE
Hotfix: Changed readme: added Python version 

### PR DESCRIPTION
**Pull request by Ren22**
+ corrected the link for pip install
Recreated PR

### CODE REVIEW

File: metaspace/python-client/README.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ Python module that provides programmatic access to a running instance of [`METAS
 
 ## Installation
 
-You need Python 3 to use this library.
+You need to have **Python 3.6 or higher** to use this library.
```
Comment (intsco) [inline metaspace/python-client/README.md]: So Python 3.6 or 3.5?

Comment (Ren22) [reply metaspace/python-client/README.md]: (even though it should work with 3.5 as well now)

Comment (LachlanStuart) [reply metaspace/python-client/README.md]: Is Sergio on 3.6 now? Because we shouldn't say 3.6 if we're actually planning to support 3.5.

Comment (Ren22) [reply metaspace/python-client/README.md]: he is on 3.5 with the changes from https://github.com/metaspace2020/metaspace/pull/140
Again, I cannot guarantee 100% support of 3.5 as I didn't test it. I personally used 3.6. If user hits the corner case again with 3.5 version, it will be our fault by misleading him with this readme

Comment (LachlanStuart) [reply metaspace/python-client/README.md]: I've experienced this before with browser versions - it causes extra work if you're not clear what is supported and what isn't. If we decide to say we only support 3.6 here, PRs like #140 shouldn't happen again - the response should just be to tell the user to upgrade. Otherwise we'll get into an annoying cycle of cleaning up code by using new features, then messing it up again to support older versions.

Comment (Ren22) [reply metaspace/python-client/README.md]: let's hope that it won't happen again, from now on we can use 3.6 syntax more freely. I'll ask Sergio to run it with 3.6 in the future

Comment (Ren22) [reply metaspace/python-client/README.md]: merging this PR

**Comment by intsco** [comment]
@Ren22 what about the branch, do you still need it?

**Comment by Ren22** [comment]
@intsco nope, I'll delete it
